### PR TITLE
Initial corrections

### PR DESCRIPTION
**Pull request by BoDonkey**
Feel free to ignore! This is really great and understandable. There is a lot to digest!

### CODE REVIEW

**Review by myovchev — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -100,29 +100,29 @@ $ npm run serve
...
+For demonstration purposes (and because by default it requires additional page injection), ReactJS is configured via its own project module `vite-react`. Looking inside the `index.js` file of that module, this is accomplished using the Apostrophe Vite `build.vite` configuration that can be added to any project module. The module also injects the React refresh runtime required for React HMR, using the new conditional injection feature within the `init(self)` block. You can read more about this in the [documentation](https://docs.apostrophecms.org/guide/vite.html#development-specific-features).
 
-Tailwind CSS is integrated site-wide and can be used in both front-end and back-end (nunjucks) code. The configuration steps used while creating the demo are described below.
+This demo also has Tailwind CSS integrated site-wide and can be used in both front-end (Nunjucks) and back-end Admin UI code. The configuration steps used while creating the demo are described below.
```
Comment (myovchev) [inline README.md]: Front-end is Admin UI and the project level UI (ui/src). Nunjucks is a backend template.

File: README.md [hunk 2]
```diff
@@ -136,28 +136,28 @@ Let's demystify the counter apps and follow their integration step by step. Ther
...
-- A simple API endpoint to save the counter value in the MongoDB database per widget instance.
-- A method to get the counter value per widget instance, used in the async server component to pass that value as a prop to the front-end app.
-- Nunjucks helpers and component to serialize server-side data and send it to the front-end app via `data-*` attributes.
+- A simple API endpoint using the Apostrophe `apiRoutes(self)` configuration method to save the counter value in the MongoDB database per widget instance.
+- A method, `getWidgetCounter(id)`, to get the counter value per widget instance, used in the async server component to pass that value as a prop to the front-end app.
+- A Nunjucks helper filter (`toAttributeValue(obj)`) and component. The component is defined in the `counterApp()` component method and the template is located in the `views` folder. In ApostropheCMS, components act much like they do in other frameworks, allowing you to add specific functionality to any of your templates. In this case, we are serializing server-side data and sending it to the front-end app via `data-*` attributes.
```
Comment (myovchev) [inline README.md]: "and a server component". I'm trying to be consistent with the term `component` which the modern dev understands as a Framework component, but in this case I've missed that.

**Review by myovchev — CHANGES_REQUESTED** [review]
Just two small details.

File: README.md (hunk 1 above)
Comment (BoDonkey) [reply README.md]: Not how I usually think about it, but I understand your reasoning. Changed.

File: README.md (hunk 2 above)
Comment (BoDonkey) [reply README.md]: Changed.

**Review by myovchev — APPROVED** [review]

**Pull request merged**